src/models/ili9225.rs

```diff
@@ -0,0 +1,250 @@
+        di.write_raw(ILI9225_GAMMA_CTRL4, &[0x00, 0x0A])?;
+        di.write_raw(ILI9225_GAMMA_CTRL5, &[0x0A, 0x08])?;
+        di.write_raw(ILI9225_GAMMA_CTRL6, &[0x08, 0x08])?;
+        di.write_raw(ILI9225_GAMMA_CTRL7, &[0x00, 0x00])?;
+        di.write_raw(ILI9225_GAMMA_CTRL8, &[0x0A, 0x00])?;
+        di.write_raw(ILI9225_GAMMA_CTRL9, &[0x07, 0x10])?;
+        di.write_raw(ILI9225_GAMMA_CTRL10, &[0x07, 0x10])?;
+
+        di.write_raw(ILI9225_DISP_CTRL1, &[0x00, 0x12])?;
+        delay.delay_us(50_000);
+        di.write_raw(ILI9225_DISP_CTRL1, &[0x10, 0x17])?;
+        delay.delay_us(50_000);
+
+        Ok(madctl)
+    }
+
+    fn update_address_window<DI>(
+        di: &mut DI,
+        rotation: Rotation,
+        sx: u16,
+        sy: u16,
+        ex: u16,
+        ey: u16,
+    ) -> Result<(), DI::Error>
+    where
+        DI: Interface,
+    {
+        match rotation {
+            Rotation::Deg0 | Rotation::Deg180 => {
+                di.write_raw(0x37, &sx.to_be_bytes())?;
+                di.write_raw(0x36, &ex.to_be_bytes())?;
+                di.write_raw(0x39, &sy.to_be_bytes())?;
+                di.write_raw(0x38, &ey.to_be_bytes())?;
+                di.write_raw(0x20, &sx.to_be_bytes())?;
+                di.write_raw(0x21, &sy.to_be_bytes())
+            }
+            Rotation::Deg90 | Rotation::Deg270 => {
+                di.write_raw(0x39, &sx.to_be_bytes())?;
+                di.write_raw(0x38, &ex.to_be_bytes())?;
+                di.write_raw(0x37, &sy.to_be_bytes())?;
+                di.write_raw(0x36, &ey.to_be_bytes())?;
+                di.write_raw(0x21, &sx.to_be_bytes())?;
+                di.write_raw(0x20, &sy.to_be_bytes())
+            }
+        }
+    }
+
+    fn sleep<DI, DELAY>(di: &mut DI, delay: &mut DELAY) -> Result<(), DI::Error>
+    where
+        DI: Interface,
+        DELAY: DelayNs,
+    {
+        di.write_raw(ILI9225_DISP_CTRL1, &[0x00, 0x00])?;
+        delay.delay_us(50_000);
+        di.write_raw(ILI9225_POWER_CTRL2, &[0x00, 0x07])?;
+        delay.delay_us(50_000);
+        di.write_raw(ILI9225_POWER_CTRL1, &[0x0A, 0x01])
+    }
+
+    fn wake<DI, DELAY>(di: &mut DI, delay: &mut DELAY) -> Result<(), DI::Error>
+    where
+        DI: Interface,
+        DELAY: DelayNs,
+    {
+        di.write_raw(ILI9225_POWER_CTRL1, &[0x0A, 0x00])?;
+        di.write_raw(ILI9225_POWER_CTRL2, &[0x10, 0x3B])?;
+        delay.delay_us(50_000);
+        di.write_raw(ILI9225_DISP_CTRL1, &[0x10, 0x17])
+    }
+
+    fn write_memory_start<DI>(di: &mut DI) -> Result<(), DI::Error>
+    where
+        DI: Interface,
+    {
+        di.write_command(WriteMemoryStartILI9225)
+    }
+
+    fn update_options<DI>(&self, di: &mut DI, options: &ModelOptions) -> Result<(), DI::Error>
+    where
+        DI: Interface,
+    {
+        options_write_cmd(di, options)
+    }
+}
+
+#[path = "../dcs/macros.rs"]
+#[macro_use]
+mod dcs_macros;
+
+dcs_basic_command!(
+    /// Software Reset
```

Comment:
Copy and paste error:
```suggestion
    /// Initiate Framebuffer Memory Write
```

Reply:
Fixed